README.md

```diff
@@ -25,111 +26,115 @@ KubePlus takes an application Helm chart and wraps it in a Kubernetes API (CRD).
 The KubePlus Operator does not need any admin-level permissions on a cluster for application providers. This allows application providers to offer their managed services on any K8s clusters including those owned by their customers. KubePlus comes with a small utility that allows you to create provider specific kubeconfig on a cluster in order to enable application deployments and management. Providers have an ability to create a consumer specific further limited kubeconfig to allow for self-service provisioning of application instances as well. 
 
 ### Resource utilization
+
 KubePlus provides controls to set per-namespace resource quotas. It also monitors usage of CPU, memory, storage, and network traffic at the application instance level. The collected metrics are available in different formats and can be pulled into Prometheus for historical usage tracking.
 
 ### Upgrades
+
 A running application instance can be updated by making changes to the spec properties of the CRD instance and applying it.
 KubePlus will update that application instance (i.e. helm upgrade of the corresponding helm release). 
 A new version of an application can be deployed by updating the application Helm chart under the existing Kubernetes CRD or registering the new chart under a new Kubernetes CRD. If the existing Kubernetes CRD object is updated, KubePlus will update all the running application instances (helm releases) to the new version of the application Helm chart.
 
 ### Customization
+
 The spec properties of the Kubernetes CRD wrapping the application Helm chart are the fields defined in the chart’s values.yaml file. Application deployments can be customized by specifying different values for these spec properties.
 
 ## Demo
 
 https://github.com/cloud-ark/kubeplus/assets/732525/efb255ff-fc73-446b-a583-4b89dbf61638
 
+## Getting Started with an example
 
+Let’s look at an example of creating a multi-instance WordPress Service using KubePlus. The WordPress service provider goes through the following steps towards this on their cluster:
 
-## Getting Started 
+1. Clone the repository locally
```

Comment:
Good addition. I suggest we also include the clone command with shallow depth:
git clone --depth 1 https://github.com/cloud-ark/kubeplus.git

This is the guidance that we have been providing in the examples since the size of the repo is big. 
Regarding the size - I will add thoughts to the issue.